tests/ImageSharp.Tests/TestImages.cs

```diff
@@ -677,6 +677,7 @@ public static class Lossy
             public const string WithXmp = "Webp/xmp_lossy.webp";
             public const string BikeSmall = "Webp/bike_lossy_small.webp";
             public const string Animated = "Webp/leo_animated_lossy.webp";
+            public const string AnimatedIssue2528 = "Webp/issues/issue2528.webp";
```

Comment:
Maybe fa4febf7fe3264a440c4d4e135dd027882e24fb0 has to be reverted now?

Reply:
yeah I think you are right. We should squash commits when merging this, because of the back and forth here